plugins/autocast/package.json

```diff
@@ -53,15 +53,15 @@
   "license": "ISC",
   "dependencies": {
     "@flatfile/hooks": "^1.3.2",
-    "@flatfile/plugin-record-hook": "^1.4.9",
     "@flatfile/util-common": "^1.3.0"
   },
   "peerDependencies": {
-    "@flatfile/api": "^1.7.4",
-    "@flatfile/listener": "^1.0.1"
+    "@flatfile/api": "^1.8.0",
+    "@flatfile/listener": "^1.0.1",
+    "@flatfile/plugin-record-hook": "^1.5.1"
   },
   "devDependencies": {
-    "@flatfile/api": "^1.7.4",
+    "@flatfile/plugin-record-hook": "^1.5.1",
```

Comment:
Shouldn't this only be in `peerDependencies`?
```suggestion
```

Reply:
No, turbo build won't realize that a plugin is dependent on another plugin unless the dependency is either in `deps` or `devDeps`.